.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -10,13 +10,12 @@ Closes #
 -
 
 ### Checklist
-- [ ] My code follows the [code-style](https://github.com/GIScience/ohsome-api/blob/master/CONTRIBUTING.md) rules, and I have checked on the [static analyses](https://jenkins.ohsome.org/job/oshdb/view/change-requests/) and [benchmark](https://reports.ohsome.org/oshdb-benchmarks/) (if applicable) results
+- [ ] My code follows the [code-style](https://github.com/GIScience/ohsome-api/blob/master/CONTRIBUTING.md) rules, and I have checked on the [static analyses](https://jenkins.ohsome.org/job/ohsome-api/view/change-requests/)
```

Comment:
Currently we don't have a `CONTRIBUTING.md` in this repository. I kept this line, because I assume we will have one, soon.

Reply:
We had the discussion once in the past whether we want to add a `CONTRIBUTING.md`, or not. We went then for restructuring + enlarging the `README.md` instead. Of course it can't be seen as a complete substitution, as e.g. info like the used Java style guide is not there.

Reply:
I think there should be one point where we want our contributors to find the (public) rules for contributing. Where to find these rules, if in the Readme or a separate file is open for discussion. I tend to separate these information as most people don't need the information on how to contribute.

Reply:
In https://github.com/GIScience/ohsome-api/pull/15 we mentioned a CONTRIBUTING.txt file, but I can't find it..? 🤔 
Not sure why we did not add it back then.. but yeah I guess it makes sense to have such a file.

Reply:
I think it is better to have two different files for contributing and readme and put in the readme a link to the contributing file.

Reply:
So, what's the best way to proceed here? Keep the CONTRIBUTING.md link in there and add the file as soon as it is written? Or do you want to drop the line and remember to add it when the file is actually there? I can also add a file copying the [oshdb CONTRIBUTING.md](https://github.com/GIScience/oshdb/blob/master/CONTRIBUTING.md).

Reply:
I made a proposal to this branch which might address this issue.

Reply:
Ah ok the CONTRIBUTING.md file in the OSHDB is also just about the code style. Yeah we can for now just take the same actually (as we have indeed the same code style) and potentially extend it in future.